keadm/cmd/keadm/app/cmd/util/common_others.go

```diff
@@ -86,7 +86,7 @@ func HasSystemd() bool {
 func RunningModuleV2(opt *types.ResetOptions) types.ModuleRunning {
 	osType := GetOSInterface()
 	cloudCoreRunning, err := IsCloudcoreContainerRunning(constants.SystemNamespace, opt.Kubeconfig)
-	if err != nil {
+	if cloudCoreRunning && err != nil {
```

Comment:
This logical expression will cause the original error message to be lost. You may consider other ways to solve this problem.

Reply:
Thank you for your review. My purpose here is to shield error logs. Cloud logs should not be output when reset is on the edge. Similarly, cloud logs should not be output on the edge. According to the original logic, when the reset operation is performed on the side, the error of the cloud must be output, but this is meaningless and will mislead the user.

Reply:
We can optimize the reset command to solve this issue, and you can look at the third task in the issue #5317